gcp/config.ts

```diff
@@ -23,29 +26,32 @@ export const akkaOperatorChartOpts: ChartOpts = {
   },
 };
 
-export function gkeNodePoolArgs(clusterName: string, zone: string | undefined): gcp.container.NodePoolArgs {
+export function gkeNodePoolArgs(
+  clusterName: pulumi.Output<string>,
+  zone: string | undefined,
+): gcp.container.NodePoolArgs {
   return {
     cluster: clusterName,
     location: zone,
-    initialNodeCount: config.getNumber("initial-node-count") || 3,
+    initialNodeCount: config.getNumber("gke.nodePool.initialNodeCount") || 3,
     autoscaling: {
-      maxNodeCount: config.getNumber("autoscaling-max-node-count") || 7,
-      minNodeCount: config.getNumber("autoscaling-min-node-count") || 1,
+      maxNodeCount: config.getNumber("gke.nodePool.autoscaling.maxNodeCount") || 7,
+      minNodeCount: config.getNumber("gke.nodePool.autoscaling.minNodeCount") || 1,
     },
     nodeConfig: {
       preemptible: true,
-      machineType: config.get<string>("node-machine-type") || "n1-standard-4",
+      machineType: config.get<string>("gke.nodePool.nodeConfig.machineType") || "n1-standard-4",
       oauthScopes: ["https://www.googleapis.com/auth/cloud-platform"],
     },
   };
 }
 
 export function databaseInstanceArgs(project: string | undefined, networkId: string): gcp.sql.DatabaseInstanceArgs {
   return {
-    databaseVersion: config.get<string>("db-version") || "POSTGRES_13",
+    databaseVersion: config.get<string>("clouSql.databaseVersion") || "POSTGRES_13",
```

Comment:
Is it meant to be `cloudSql` instead of `clouSql`?

Reply:
Fixed: 94918d0dcb80525bcc26bfaf2aa2ac7487b024fd. 

Good eye, btw.